cli/command/container/cp.go

```diff
@@ -34,11 +37,42 @@ const (
 type cpConfig struct {
 	followLink bool
 	copyUIDGID bool
+	quiet      bool
 	sourcePath string
 	destPath   string
 	container  string
 }
 
+//cpWithProgress is a Wrapper to the content being copied from or to container. This wrapper
+//contains information about the file or folder being copied (size and bytes - ReadCloser) as well as
+//source and destination path. When the bytes are copied, the `Read` method from io.ReadCloser is overwritten
+//showing the progress during the copy.
+type cpWithProgress struct {
+	io.ReadCloser
+	toContainer   bool
+	total         *int64
+	containerName string
+	copyPath      string
+	quiet         bool
+	writer        io.Writer
+}
+
+func (pt *cpWithProgress) Read(p []byte) (int, error) {
+	n, err := pt.ReadCloser.Read(p)
+	*pt.total += int64(n)
+
+	if err == nil && !pt.quiet {
+		fmt.Fprint(pt.writer, "\033[u\033[K")
```

Comment:
Also see my other comment/question; I _think_ this is "erase line", correct? If so, we could replace this with;

```suggestion
		fmt.Fprint(pt.writer, aec.EraseLine(aec.EraseModes.All))
```

(but double check if I'm correct 😅)

Reply:
Yeah, erase line.